specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -0,0 +1,2684 @@
+          }
+        }
+      }
+    },
+    "CloudServiceOsProfile": {
+      "description": "Describes the OS profile for the cloud service.",
+      "type": "object",
+      "properties": {
+        "secrets": {
+          "description": "Specifies set of certificates that should be installed onto the role instances.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceVaultSecretGroup"
+          }
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfigurationProperties": {
+      "description": "Describes a cloud service IP Configuration",
+      "type": "object",
+      "properties": {
+        "publicIPAddress": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the public ip address resource."
+        },
+        "subnet": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the virtual network subnet resource."
+        },
+        "privateIPAddress": {
+          "description": "The virtual network private IP address of the IP configuration.",
+          "type": "string"
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfiguration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the resource that is unique within the set of frontend IP configurations used by the load balancer. This name can be used to access the resource.",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerFrontendIPConfigurationProperties",
+          "description": "Properties of load balancer frontend ip configuration."
+        }
+      },
+      "required": [
+        "name",
+        "properties"
+      ]
+    },
+    "LoadBalancerConfigurationProperties": {
+      "type": "object",
+      "properties": {
+        "frontendIPConfigurations": {
+          "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+          }
+        }
+      },
+      "required": [
+        "frontendIPConfigurations"
+      ]
+    },
+    "LoadBalancerConfiguration": {
+      "description": "Describes the load balancer configuration.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource Id",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the Load balancer",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerConfigurationProperties",
+          "description": "Properties of the load balancer configuration."
+        }
+      },
+      "required": [
```

Comment:
Adding a required property to an existing object is a breaking change. We do not want to have any breaking changes comparing the latest Preview release with the first Stable release.
I think you're also trying to edit this in the preview version. That Preview needs to release and release the SDK before this change can be merged.

Reply:
resolved as per email discussion